### PR TITLE
Updating the npm instructions to pull from our Ping Identity npm.

### PR DESCRIPTION
**Pull request by philipcoldrey-ping**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Note: A 'flowId' value is required for the widget to interact with PingFederate,
 This is a good choice if you already have a node project with `package.json` and can just add this widget as a dependency.
 
 To add the widget as a dependency:
-1. Run `npm install pf-authn-widget --save`
+1. Run `npm install @ping-identity/pf-authn-js-widget --save`
```
Comment (philipcoldrey-ping) [inline README.md]: @khosravian I believe this is the only instruction that needs updating. Let me know if there is more and I will make the change.

Comment (khosravian) [reply README.md]: O this doesn't need to change

Comment (khosravian) [reply README.md]: `authnwidget` is correct

Comment (philipcoldrey-ping) [reply README.md]: Oh, thanks. Let me back that out.

**Comment by khosravian** [comment]
looks good

**Pull request merged**